lib/index.js

```diff
@@ -2372,6 +2371,41 @@ Stream.prototype.take = function (n) {
 exposeMethod('take');
 
 /**
+ * Like take(n), only instead of returning the first n values, it ignores the first n values and emits the rest
```

Comment:
Period at the end of this sentence.

Also, I wouldn't call `drop` "like take". More like

The opposite of [take](#take). Ignore the first `n` values and emit the rest. All errors (even ones emitted before the nth value) will be emitted.